packages/forma-36-react-components/src/components/Asset/README.mdx

```diff
@@ -0,0 +1,144 @@
+import { Asset } from './Asset';
+
+# Asset
+
+The Asset component is a visual representation of an asset file such as a video or a PDF document.
+
+<Asset title="Archive of files" type="archive" />
+
+## Table of contents
```

Comment:
I will add `Table of contents` to other README's if we want to have it (I think it's a good idea). And I will update the template as well.